db/migrations/20231017005257_add_origin_weight.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+-- modify "origins" table
+ALTER TABLE "origins" ADD COLUMN "weight" bigint NOT NULL DEFAULT 100;
```

Comment:
`bigint` too large for the weights? Maybe `smallint`?

Reply:
seems like using smallint/int16 requires some manual implementations so I used int/int32 instead